docs/Learning-Environment-Design.md

```diff
@@ -66,32 +65,27 @@ information.
 ## Organizing the Unity Scene
 
 To train and use the ML-Agents toolkit in a Unity scene, the scene must contain
-a single Academy subclass and as many Agent subclasses
-as you need.
+a single Academy and as many Agent subclasses as you need.
 Agent instances should be attached to the GameObject representing that Agent.
 
 ### Academy
 
 The Academy object orchestrates Agents and their decision making processes. Only
 place a single Academy object in a scene.
 
-You must create a subclass of the Academy class (since the base class is
-abstract). When you create your Academy subclass, you can implement the
-following methods (all are optional):
-
-* `InitializeAcademy()` — Prepare the environment the first time it launches.
-* `AcademyReset()` — Prepare the environment and Agents for the next training
-  episode. Use this function to place and initialize entities in the scene as
-  necessary.
-* `AcademyStep()` — Prepare the environment for the next simulation step. The
-  base Academy class calls this function before calling any `AgentAction()`
-  methods for the current step. You can use this function to update other
-  objects in the scene before the Agents take their actions. Note that the
-  Agents have already collected their observations and chosen an action before
-  the Academy invokes this method.
-
-See [Academy](Learning-Environment-Design-Academy.md) for a complete list of
-the Academy properties and their uses.
+#### Academy resetting
```

Comment:
Moved this section from docs/Learning-Environment-Design-Academy.md  (now defunct)